daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SExpr.scala

```diff
@@ -38,7 +38,7 @@ object SExpr {
     * that is, SEVar(1) points to the top-most value in the environment.
     * https://en.wikipedia.org/wiki/De_Bruijn_index
     */
-  final case class SEVar(index: Int) extends SExpr {
+  final case class SVar(index: Int) extends SExpr {
```

Comment:
I'm generally not a big fan of doing a renaming like this and actual changes in the same PR. I think I understand why you want to rename it but it might be better to the in separate PR before the one with the actual changes.

Reply:
I don't like this at all. It conflicts with the naming convention used for SValue. Please change it back.

Reply:
I created #2657 to address this comment. 
I will switch the PR to ready to review once #2657 is merged.